packages/devextreme/js/localization/messages/en.json

```
@@ -55,6 +55,7 @@
         "dxList-selectAll-checked": "Checked",
         "dxList-selectAll-notChecked": "Not checked",
         "dxList-ariaRoleDescription": "List",
+        "dxList-listAriaLabel-itemContent": "List Item Content",
```

```suggestion
        "dxList-listAriaLabel-itemContent": "List item content",
```

Please also replace in other locales

---

Fixed